examples/android/app/src/main/java/io/grpc/helloworldexample/HelloworldActivity.java

```diff
@@ -47,7 +46,7 @@ public void sendMessage(View view) {
         private String mHost;
         private String mMessage;
         private int mPort;
-        private ChannelImpl mChannel;
+        private ManagedChannel mChannel;
```

Comment:
Hmm ... did it compile before? There was previously no import for `ChannelImpl` either.  Done.